src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -1013,11 +1014,18 @@ fn preprocess_link(
     // | has backtick |    never ignore    |    never ignore   |
     // | no backtick  | ignore if url-like |    never ignore   |
     // |-------------------------------------------------------|
-    let ignore_urllike =
-        can_be_url || (ori_link.kind == LinkType::ShortcutUnknown && !ori_link.link.contains('`'));
+    let ignore_urllike = can_be_url || (is_shortcut_style && !ori_link.link.contains('`'));
     if ignore_urllike && should_ignore_link(path_str) {
         return None;
     }
+    // ignore github flavored markdown special blockquotes,
+    // such as [!NOTE] and [!IMPORTANT]
```

Comment:
Could you elaborate the comment slightly noting that it's ~common for people to include_str the README.md of their GitHub repo?

I think we want to highlight that cuz rustdoc doesn't use GitHub-flavored Markdown and doesn't support GitHub-style admonitions.

Reply:
yeah, good point, will elaborate.